### PR TITLE
Update bitfields.md

### PR DESCRIPTION
**Pull request by sav-norem**
update bitfields data type page to include hugo short code - condense example and add relevant context

### CODE REVIEW

**Comment by sav-norem** [comment]
@dwdougherty bitfields for your viewing pleasure

File: docs/data-types/bitfields.md [hunk 1]
```diff
@@ -21,37 +21,27 @@ Bitfields support atomic read, write and increment operations, making them a goo
 
 ## Examples
```
Comment (dwdougherty) [inline docs/data-types/bitfields.md:22]: ## Example

File: docs/data-types/bitfields.md [hunk 2]
```diff
@@ -21,37 +21,27 @@ Bitfields support atomic read, write and increment operations, making them a goo
...
 
-Suppose you're keeping track of activity in an online game.
-You want to maintain two crucial metrics for each player: the total amount of gold and the number of monsters slain.
-Because your game is highly addictive, these counters should be at least 32 bits wide.
+Suppose you're keeping track of stats for various bicycles.
+You want to maintain two crucial metrics for each bike: the current price and the number of owners.
+For this example, we're making the counters 32 bits wide.
 
 You can represent these counters with one bitfield per player.
```
Comment (dwdougherty) [inline docs/data-types/bitfields.md:28]:
```suggestion
Suppose you want to maintain two metrics for various bicycles: the current price and the number of owners over time. You can represent these counters with a 32-bit wide bitfield per for each bike.
```

File: docs/data-types/bitfields.md [hunk 3]
```diff
@@ -21,37 +21,27 @@ Bitfields support atomic read, write and increment operations, making them a goo
...
 
-* New players start the tutorial with 1000 gold (counter in offset 0).
-```
-> BITFIELD player:1:stats SET u32 #0 1000
-1) (integer) 0
-```
+* Bike 1 initially costs 1,000 (counter in offset 0) and has never had an owner. After being sold, it's now considered used and the price instantly drops to reflect its new condition, and it now has an owner (offset 1). After quite some time, the bike becomes a classic. The original owner sells it for a profit, so the price goes up and the number of owners does as well. Lastly, we'll look at the bikes current price and number of owners.
```
Comment (dwdougherty) [inline docs/data-types/bitfields.md:30]:
```suggestion
* Bike 1 initially costs 1,000 (counter in offset 0) and has never had an owner. After being sold, it's now considered used and the price instantly drops to reflect its new condition, and it now has an owner (offset 1). After quite some time, the bike becomes a classic. The original owner sells it for a profit, so the price goes up and the number of owners does as well. Finally, you can look at the bike's current price and number of owners.
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
A few edits.

**Comment by sav-norem** [comment]
@dwdougherty updated with feedback

File: docs/data-types/bitfields.md [hunk 4]
```diff
@@ -21,37 +21,25 @@ Bitfields support atomic read, write and increment operations, making them a goo
...
+1) (integer) 950
 2) (integer) 1
-```
+> BITFIELD bike:1:stats INCRBY u32 #0 500 INCRBY u32 #1 1
+1) (integer) 1450
+2) (integer) 2
+> BITFIELD bike:1:stats GET u32 #0 GET u32 #1
+1) (integer) 1450
+2) (integer) 2
+{{< /clients-examples >}}
```
Comment (dwdougherty) [inline docs/data-types/bitfields.md:42]:
```suggestion
{{< /clients-example >}}
```

**Pull request merged**